### ISSUE TITLE
Enable pagination of games

### ISSUE THREAD

**Issue by sgstair** [ludumdare/ludumdare#1324]
A common request is that, instead of using the feed for games, we should have a paged view.

I'm aware that the intent of the feed is to be more mobile-friendly, so we might want to add a dedicated games list for the event that can be paginated. If you have any other ideas please add them here.

**Comment by MeiHouwang** [comment]
I want to repeat my point there as I did in LD feed:
In my opinion(and I don't think it is only mine) we badly need proper paged game list as it was on the old ludumdare.com. There are many points to it:
- The feed is uncomfortable to use. Every single human I've talked about it found it uncomfortable and counter-intuitive.
- It makes extremely hard to look up for games at the middle of the list. Spending half a hour clicking "Load More" and scroll through entries that you already saw and want just skip over? I don't find it funny.
- With the lack of search functionality the current implementation makes it so that many entries will never be seen at all. There are plenty of creators who does not care about socialization and just make game and upload it. And some of them create pretty interesting and original games that is seen only by several people at the beginning and after that linger forever at the middle of the list, not showing with any sort settings due to low karma and the fact that most people doesn't go futher than dozen pages. I for one liked very much to search such hidden gems in the middle of the list. Now I can't do it, sob.
- Please correct me if I wrong but I don't quite understand the point of making list mobile-friendly. Especially by the means of making it PC-unfriendly. The vast majority of the games on LD are downlodable PC builds. The main target platform of jammers is still PC. While nobody restricts making mobile or web games, this is not a mobile jam and mobile platforms are minority. So the main use of the game list at the current moment is: Browsing entries on your PC, downloading builds, and play them still on your PC. I really doubd that someone browse entries on mobile device just to find and dl them later on PC(since it is really bothersome and sometimes impossible task to find some specific game in the game feed). So tell me please, why are we hindering the actual jam participants and make it harder for them to browse and rate games? Is it only to make the third parties life easier?
- I actually tried to load and scroll though game feed on mobile device(iPad). Surprisingly it was even worse than on PC browser. Again, old list on ludumdare.com is way more convinient to use even on the mobile device. So I'm at a little loss now - why the hell do we need that game feed at all???

**Comment by povrazor** [comment]
There was a pagination effort at some point (Possibly by @Noojuno?). Not sure what happened to that though.

**Comment by Noojuno** [comment]
Yeah, I started with pagination, but then we decided to actually properly plan out the feed endpoint before I did it how I wanted to :P It's currently number 3 on my `//TODO:` list, so pretty soon I'll be taking a look at it.

**Comment by sgstair** [comment]
I feel like hacking it in to the current feed (probably add a new query term for "get count") is a good idea for LD40, it won't be hard to refactor when we rework the feeds. I can do the backend part.

**Comment by sgstair** [comment]
@MeiHouwang I'm curious - What is your ping to the site? (ldjam.com or api.ldjam.com), I expect it's pretty significant, I'd like to simulate the additional ping to see what we can do to improve performance in the short term.
